tensorflow/compiler/xla/service/gpu/gemm_rewriter.cc

```diff
@@ -187,18 +203,55 @@ class GemmRewriterVisitor : public DfsHloRewriteVisitor {
   }
 
   Status HandleAdd(HloInstruction *instr) override {
-    HloInstruction *bias, *existing_gemm;
+    HloInstruction *bias, *existing_gemm, *slice;
 
-    // First, try to match vector bias add, so we might elide the broadcast.
+    // Attempt to elide broadcast and fuse addition of a vector bias into GEMM.
     if (Match(instr, m::AddAnyOrder(
                          m::Op(&existing_gemm)
                              .WithCustomCallTarget(kCublasLtMatmulCallTarget),
                          m::Broadcast(&bias, m::Op())))) {
       TF_ASSIGN_OR_RETURN(bool was_fused,
-                          FuseVectorBiasAdd(instr, bias, existing_gemm));
+                          FuseVectorBiasAddSliced(instr, bias, existing_gemm));
```

Comment:
I'm confused by this new function name.  This is the case where it's not sliced, but the function name says "Sliced".  Why?

Reply:
The name was changed to indicate that the function can replace the matched instruction with a slicing instruction instead of a GEMM.

Reply:
But isn't it confusing to call a function called "FuseFooSliced" in the case when there are no slices involved?

FuseFooMaybeSliced if you really want "sliced" in the name?

Or tbh `FuseFoo(x, y, /*slice=*/z)` seems pretty clear?

Reply:
MaybeSliced is the clearest option, but I agree that Sliced could also be dropped from the names.